examples/diabetes_example/run_diabetes.py

```diff
@@ -0,0 +1,500 @@
+                )
+            },
+            worker=client_id,
+        )
+    )
+    for client_id in DATA_PROVIDER_ORGS_ID
+}
+
+# %%
+# In local_compute mode, the registered task is executed at once:
+# the registration function returns a value once the task has been executed.
+#
+# In deployed mode, the registered task is added to a queue and treated asynchronously: this means that the
+# code that registers the tasks keeps executing. To wait for a task to be done, create a loop and get the task
+# every n seconds until its status is done or failed.
+#
+
+aggregation_1_inputs = [
+    InputRef(
+        identifier="local_analytics_list",
+        parent_task_key=local_key,
+        parent_task_output_identifier="local_analytics_first_moments",
+    )
+    for local_key in local_task_1_keys.values()
+]
+
+
+aggregation_task_1 = TaskSpec(
+    function_key=aggregate_function_key,
+    inputs=aggregation_1_inputs,
+    outputs={"shared_states": ComputeTaskOutputSpec(permissions=permissions)},
+    worker=ALGO_ORG_ID,
+)
+
+aggregation_task_1_key = clients[ALGO_ORG_ID].add_task(aggregation_task_1)
+
+
+# %%
+
+shared_inputs = [
+    InputRef(
+        identifier="shared_states",
+        parent_task_key=aggregation_task_1_key,
+        parent_task_output_identifier="shared_states",
+    )
+]
+
+local_task_2_keys = {
+    client_id: clients[client_id].add_task(
+        TaskSpec(
+            function_key=local_function_2_keys[client_id],
+            inputs=data_manager_input[client_id]
+            + datasample_inputs[client_id]
+            + shared_inputs,
+            outputs={
+                "local_analytics_second_moments": ComputeTaskOutputSpec(
+                    permissions=permissions
+                )
+            },
+            worker=client_id,
+        )
+    )
+    for client_id in DATA_PROVIDER_ORGS_ID
+}
+
+
+aggregation_2_inputs = [
+    InputRef(
+        identifier="local_analytics_list",
+        parent_task_key=local_key,
+        parent_task_output_identifier="local_analytics_second_moments",
+    )
+    for local_key in local_task_2_keys.values()
+]
+
+aggregation_task_2 = TaskSpec(
+    function_key=aggregate_function_key,
+    inputs=aggregation_2_inputs,
+    outputs={"shared_states": ComputeTaskOutputSpec(permissions=permissions)},
+    worker=ALGO_ORG_ID,
+)
+
+aggregation_task_2_key = clients[ALGO_ORG_ID].add_task(aggregation_task_2)
+
+
+# %%
+# Results
+# -------
+# Now we can view the results.
+#
+
+import pickle
+
+task1 = clients[ALGO_ORG_ID].get_task(aggregation_task_1_key)
+task2 = clients[ALGO_ORG_ID].get_task(aggregation_task_2_key)
+
+with open(task1.outputs["shared_states"].value.address.storage_address, "rb") as f:
+    out1 = pickle.load(f)
+with open(task2.outputs["shared_states"].value.address.storage_address, "rb") as f:
+    out2 = pickle.load(f)
+
+print(
+    f"""Age mean: {out1['means']['age']}
+Sex percentage: {out1['counts']['sex']}
+Blood pressure std: {out2["std"]["bp"]}
+"""
```

Comment:
For better readability, I would limit to 2 number after the coma, and add units (years, %, bp std unit...)